dev/excluded_copyright_files.txt

```diff
@@ -0,0 +1,3 @@
+*backends-clickhouse/*
```

Comment:
Hi @zzcclp, this PR introduces a license header checker for source code file. And a CI job will do the check henceforth. Here, CH modules are excluded. You can file another PR for including CH if you think necessary. Thanks!